src/lang/po/es/Prusa-Firmware-Buddy_es.po

```diff
@@ -116,41 +116,41 @@ msgstr "Cambiar Filamento"
 #: src/common/bsod.cpp:232
 msgid "Check the heatbed heater & thermistor wiring for possible damage."
 msgstr ""
-"Revisa el cableado del termistor y el calentador de la base para detectar "
+"Revise el cableado del termistor y el calentador de la base para detectar "
 "posibles daños."
 
 #: src/common/bsod.cpp:233
 msgid "Check the heatbed thermistor wiring for possible damage."
 msgstr ""
-"Revisa el cableado del termistor de la base calefactada buscando posibles "
+"Revise el cableado del termistor de la base calefactada para detectar posibles "
 "daños."
 
 #: src/common/bsod.cpp:234
 msgid "Check the print head heater & thermistor wiring for possible damage."
 msgstr ""
-"Revisa el cableado del termistor y del calentador del fusor para ver si hay "
+"Revise el cableado del termistor y del calentador del extrusor para ver si hay "
```

Comment:
You are wrongly assuming hotend==extruder. The extruder is the mechanism that pushes filament into the hotend. Hotend is where the filament is melted and deposited through the nozzle.

Reply:
I also think it is weird to call it fusor but in any case the perfect translation would be "punta caliente"